contentcuration/contentcuration/viewsets/file.py

```diff
@@ -156,6 +157,9 @@ def upload_url(self, request):
         retval = get_presigned_upload_url(
             filepath, checksum_base64, 600, content_length=size
         )
+        duration = request.data.get("duration")
+        if duration is not None:
+            duration = math.ceil(duration)
```

Comment:
This would still raise a 500 with the value in the issue, doing `math.ceil('175.033000\n')` gives a `TypeError` - probably need to coerce to a float first `duration = math.ceil(float(duration))` - and do in a try except for `ValueError` to then raise a 400 if it's not coerceable?

Reply:
If it's a string, I will just make it invalidated. This should be a number

Reply:
Sure - can just catch the TypeError then!